ci/blackbox/rdsbroker_test.go

```diff
@@ -253,106 +275,105 @@ var _ = Describe("RDS Broker Daemon", func() {
 		})
 	})
 
+	var _ = Describe("MySQL Final snapshot enable/disable", func() {
```

Comment:
This indentation change looks unrelated to the added tests. Could it be in a separate commit?

Also, the `var _ = ` is unnecessary (it's only needed at the top level).